packages/health-check/Dockerfile

```diff
@@ -1,16 +1,16 @@
 FROM node:16.14.2-alpine
 
-RUN apk update && apk add dnsmasq
+RUN apk add --no-cache dnsmasq=2.86-r0 && \
```

Comment:
Isn't the point of using `--no-cache` _not_ having to remove the cache afterwards?

Reply:
right, fixed, thanks!